### PR TITLE
Clarify JVMPORT030W and add message when core_pattern starts with a pipe

### PR DESCRIPTION
**Pull request by kgibm**
Issue: #16367

Also changed the `ABRT` sample as that has been [deprecated](https://access.redhat.com/documentation/pt/red_hat_enterprise_linux/8/html/8.6_release_notes/deprecated_functionality) in favor of `systemd-coredump`.

### CODE REVIEW

File: runtime/nls/port/port.nls
Comment (pshipton) [inline runtime/nls/port/port.nls:342]: By changing an existing message, only the English output will have this change and other languages will not until the next translation cycle next year. I may be better to add a new message that will output in English for all locales until it's translated.

Comment (kgibm) [reply runtime/nls/port/port.nls:342]: If I understand correctly, do you mean add a new message and use that in place of JVMPORT030W? If so, I worry that some users might be watching for JVMPORT030W as a symptom of a core dump and that check would no longer work.

Comment (keithc-ca) [reply runtime/nls/port/port.nls:342]: There is a reference in `J9NLS_VM_CHECK_DUMP_COREPATTERN_PIPE.system_action` that should be updated.

**Review by pshipton — APPROVED** [review]

File: runtime/nls/port/port.nls
Comment (keithc-ca) [inline runtime/nls/port/port.nls:347]: This is the other way around: `core_pattern` is how you configure a system to use `systemd-coredump`; I don't think the tool consults that file (it only considers the command-line arguments it gets as specified in `core_pattern`).

**Review by keithc-ca — CHANGES_REQUESTED** [review]

File: runtime/nls/port/port.nls
Comment (kgibm) [reply runtime/nls/port/port.nls:347]: Good point. How about this?

> /proc/sys/kernel/core_pattern starting with | is used to redirect the core file to tools such as systemd-coredump which processes the core dump, for example to send it to a different directory and manage total core dump disk usage, or to suppress it.

File: runtime/nls/port/port.nls
Comment (kgibm) [reply runtime/nls/port/port.nls:506]: omr depends on the definition of this message in openj9. For example, here is the usage of `J9NLS_PORT_LINUXDUMP_PIPE_CORE`: https://github.com/eclipse/omr/blob/ef4d57d/port/linux/omrosdump_helpers.c

So I presume that since openj9 depends on omr, that once J9NLS_PORT_LINUXDUMP_PIPE_CORE_NOT_FOUND is fully merged, then omr can be updated to use this new message. I guess something must generate the following IDs:

```
$ grep -r J9NLS_PORT_LINUXDUMP_PIPE_CORE *
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE__MODULE 0x504f5254
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE__ID 30
nls/portnls.h:#define J9NLS_PORT_LINUXDUMP_PIPE_CORE J9NLS_PORT_LINUXDUMP_PIPE_CORE__MODULE, J9NLS_PORT_LINUXDUMP_PIPE_CORE__ID
port/linux/omrosdump_helpers.c:			portLibrary->nls_printf(portLibrary, J9NLS_WARNING | J9NLS_STDERR, J9NLS_PORT_LINUXDUMP_PIPE_CORE, corePatternFileName, corePatternFormat, pid);
```

Comment (pshipton) [reply runtime/nls/port/port.nls:506]: The java.properties file contains the following
```
PORT030=%s setting "%s" specifies that the core dump is to be piped to an external program.  Attempting to rename either core or core.%d.\n
```

Once this is merged you can look there to find the new number (which looks like it will be 49).

**Comment by keithc-ca** [comment]
Updated the description so it won't close the related issue; that should wait until the remaining changes (in OMR) have been merged.

**Review by keithc-ca — APPROVED** [review]

**Comment by keithc-ca** [comment]
Jenkins compile amac jdk17

**Comment by kgibm** [comment]
No particular rush for me, but I will work on building and testing and opening the OMR issue/PR today and hopefully we can get it in if all goes well.

**Comment by kgibm** [comment]
Matching OMR Issue https://github.com/eclipse/omr/issues/6837 and PR https://github.com/eclipse/omr/pull/6838 opened

**Pull request merged**
